mycroft/client/enclosure/mark1/__init__.py

```diff
@@ -193,7 +198,8 @@ def process(self, data):
 
             LOG.info("Setting opt_in to: " + word)
             new_config = {'opt_in': enable}
-            user_config = LocalConf(USER_CONFIG)
+            user_config = LocalConf(os.path.join(
+                BaseDirectory.save_config_path('mycroft'), 'mycroft.conf'))
```

Comment:
could the USER_CONFIG vaariable be used again now that it's defined to the as the save_config_path?